addons/mail/static/src/core/common/composer_model.js

```diff
@@ -37,6 +40,7 @@ export class Composer extends Record {
     mentionedChannels = Record.many("Thread");
     cannedResponses = Record.many("CannedResponse");
     textInputContent = "";
+    fullFeature = true;
```

Comment:
Poor named field, as I said in a previous comment. This should be named `portal=true/false` instead, so we know it's for portal-specific things.

If you find a better name that is more meaningful for what it does, then I don't mind using this other name. But if you don't know of a better name, I would choose `portal=true/false` for the time being, as a small increase of technical debt that we'll repay at a later date when we are more inspired of a good name and/or better wording of the underlying feature portal chatter differ from usual chatter for the composer